src/plugins/intel_npu/src/plugin/src/backends.cpp

```diff
@@ -139,6 +139,22 @@ std::string NPUBackends::getBackendName() const {
     return "";
 }
 
+uint32_t NPUBackends::getDriverVersion() const {
+    if (_backend != nullptr) {
+        return _backend->getDriverVersion();
+    }
+
+    return 0;
```

Comment:
It is unlikely to happen that we end up in returning `0`, but if so, can you consider throwing an exception instead to signal that something goes wrong?

Reply:
Actually, there is one more layer over this, in Metrics which throws an exception in case the backend is empty. The method from metrics is used for getting the property.

Reply:
So there is no need for condition, just turn in into assertion?

Reply:
done